portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
 /**
- * Adds a random greeting to the page.
+* Picks a Spotify song to display.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function pickSpotifySong() {
+    var ids =
```

Comment:
Consider making this a constant that exists outside the function. No need to recreate this array every time you want a new song